### DISCUSSION TITLE
CSS Houdini supports

### DISCUSSION THREAD

**Discussion by piaskowyk** [facebook/react-strict-dom#156] [General]
Hey! Do you have any plans for the future to supports new CSS API like [Houdini](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Houdini) and [Painting](https://developer.mozilla.org/en-US/docs/Web/API/CSS_Painting_API)?

**Comment by nmn** [comment]
Most of Houdini is only implemented in Chrome and the entire effort has mostly lost steam so I don't think it's worthwhile spending any time bringing that to RSD.

The one feature that came out of Houdini that *might* make sense in RSD is typed CSS variables through `@property` features. StyleX recently added the ability to define variables with types and it might make sense to support the same syntax on RN as an extra layer of validation.

In modern browsers, typed variables can often also be animatable. It *might* be possible to use `Animated` on the RN to make these variables animatable on RN as well.

**Reply by piaskowyk** [reply]
Thanks for the clarification!
